tests/unit/test_project.py

```diff
@@ -84,12 +84,16 @@ def yaml_loaded_data():
 def pebble_part():
     return {
         "pebble": {
-            "plugin": "go",
+            "plugin": "nil",
             "source": "https://github.com/canonical/pebble.git",
-            "build-snaps": ["go/1.19/stable"],
+            "override-pull": (
+                "craftctl default\n"
+                "snap set system experimental.parallel-instances=true\n"
+                "snap install go_pebble --channel 1.19/stable --classic\n"
```

Comment:
I see, so `go_pebble` means an instance of the `go` snap called `pebble`?
I found this reference, for whoever else was unaware for parallel instances: https://snapcraft.io/docs/parallel-installs